configs/nuxt3.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+  extends: [require.resolve('./vue3')],
+  overrides: [
+    {
+      files: [
+        '**/pages/**/*.{js,ts,vue}',
+        '**/layouts/**/*.{js,ts,vue}',
+        '**/app.{js,ts,vue}',
+        '**/error.{js,ts,vue}',
+      ],
+      rules: { 'vue/multi-word-component-names': 'off' },
+    },
+    {
+      files: ['*.vue'],
+      rules: {
+        'no-undef': 'off',
```

Comment:
idem, waarom hier off?

Reply:
Omdat Nuxt 3 heel veel auto imports heeft. Dit zorgt ervoor dat je bijv. `const { $apiClient } = useNuxtApp()` kan doen zonder `useNuxtApp` te importeren. Eslint geeft dan een onterechte 'no-undef' error.

Reply:
Dit is wel een beetje jammer, omdat je hiermee ook daadwerkelijke vergeten imports etc. maskeerd

Reply:
Klopt. Dit is niet zo in TypeScript files omdat de check daar wel weer goed werkt. Of we moeten explicit imports gaan gebruiken https://nuxt.com/docs/guide/concepts/auto-imports#explicit-imports.

Reply:
Mjah, ik weet het niet zo goed. Laten we "gewoon" zsm typescript gebruiken ;)